components/atom/button/src/index.scss

```diff
@@ -0,0 +1,180 @@
+@import '~@schibstedspain/sui-theme/lib/settings';
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$bgc-button: $c-primary !default;
+$bgc-button-light: color-variation($bgc-button, 4) !default;
+$bgc-button-dark: color-variation($bgc-button, -1) !default;
+$bgc-button-light-contrast: color-variation($bgc-button, 3) !default;
+
+$sz-button-small: $p-base*4 !default;
+$fz-button-small: $fz-s !default;
+$icon-sz-button-small: $sz-icon-s !default;
+$icon-m-button-small: $m-s !default;
+$p-button-small: $p-m !default;
+
+$sz-button: $p-base*5 !default;
+$fz-button: $fz-m !default;
+$icon-sz-button: $sz-icon-s !default;
+$icon-m-button: $m-m !default;
+$p-button: $p-l !default;
+
+$sz-button-large: $p-base*6 !default;
+$fz-button-large: $fz-m !default;
+$icon-sz-button-large: $sz-icon-m !default;
+$icon-m-button-large: $m-m !default;
+$p-button-large: $p-l !default;
+
+// Set all color properties for text and icons
+@mixin button-foreground-color ($color) {
+  color: $color;
+
+
+  & svg path {
+    fill: $color;
+    stroke: $color;
+  }
+}
+// Sets active states when button is not disabled
+@mixin button-focused {
+  &:not(.sui-AtomButton--disabled) {
+    &.sui-AtomButton--focused,
+    &:active,
+    &:focus,
+    &:hover {
+      @content;
+    }
+  }
+}
+@mixin button-icon ($size, $margin) {
+  & .sui-AtomButton-leftIcon,
+  & .sui-AtomButton-rightIcon {
+    height: $size;
+    width: $size;
+  }
+
+  & .sui-AtomButton-leftIcon {
+    margin-right: $margin;
+  }
+
+  & .sui-AtomButton-rightIcon {
+    margin-left: $margin;
+  }
+}
+
+.sui-AtomButton {
+  @include reset-button;
+  @include button-icon($icon-sz-button, $icon-m-button);
+  border: 1px solid;
+  border-radius: $bdrs-m;
+  box-sizing: border-box;
+  font: {
+    family: $ff-sans-serif;
+    size: $fz-button;
+    weight: $fw-semi-bold;
+  }
+  height: $sz-button;
+  min-width: $sz-button;
+  outline: 0;
+  padding: 0 $p-button;
+  transition: background-color .3s ease, border-color .3s ease;
```

Comment:
For a consistent animations, i think we could define a timing variable in sui-theme.

Reply:
I proposed it to designers and they said they'll treat the animation thing later on.
Actually the transition was not defined by them. I just did it because I saw it in the IJ buttons. But maybe I should remove it as it was not clearly defined. What do you think?

Reply:
Looks fine to me, but i would define a global variable:

```
$trsdu-base: .3;
$trsdu-button: $trsdu-base !default;
```

So every vertical can assign their own speed.
IMHO, it's the same case as we should manage opacity.

Reply:
Removing animation as not defined by design.